tests/test_arguments.py

```diff
@@ -280,7 +280,7 @@ def test_valid_argument_combinations_accepted(chosen_arguments):
     ],
 )
 def test_invalid_argument_combinations_accepted(chosen_arguments):
-    """Check that not valid argument combinations do not verify correctly"""
+    """Check that not valid argument combinations do not verify correctly.."""
```

Comment:
There are some duplicate periods probably introduced by find-and-replace. It can probably be fixed with more!

Reply:
Thanks for noticing this mistake, I have fixed it @Michionlion.